_data/packages.yml

```diff
@@ -1,18 +1,57 @@
+- package_name: THzTools
+  package_description:
+  submitting_author:
+      name: Name
+      github_username: jsdodge
+  all_current_maintainers:
+    - name: J. Steven Dodge
+      github_username: jsdodge
+  repository_link: https://github.com/dodge-research-group/thztools
+  version_submitted: 0.5.4
+  categories:
+    - data-processing-munging
+  editor:
+      name: Bane Sullivan
+      github_username: banesullivan
+  eic:
+      name:
+      github_username: cmarmo
+  reviewers:
+    - name: Frank Male
+      github_username: frank1010111
+    - name:
+      github_username: Romain-Peretti
+  archive: TBD
+  version_accepted: TBD
+  date_accepted: missing
+  created_at: 2024-08-01 11:54:53+00:00
+  updated_at: 2024-11-20 00:07:21+00:00
+  closed_at:
+  issue_link: https://github.com/pyOpenSci/software-submission/issues/209
+  joss:
+  partners:
+  gh_meta:
+      name: thztools
+      description: Tools for terahertz time-domain spectroscopy (THz-TDS)
+      created_at: '2022-11-22'
+      stargazers_count: 10
+      watchers_count: 10
+      open_issues_count: 2
+      forks_count: 1
+      documentation: https://dodge-research-group.github.io/thztools/
+      contrib_count: 5
+      last_commit: '2024-11-19'
+  labels:
+    - 6/pyOS-approved
 - package_name: Stingray
   package_description: A spectral-timing software package for astrophysical X-ray
       (and other) data
   submitting_author:
-      name: Matteo Bachetti
+      name: Name
```

Comment:
so this is actually coming from the review. Name is listed as a word in the actual review. i'm just going to remove it as it's a typo and  not something worth spending a ton of time troubleshooting. every once in a while we just need to do a small cleanup. i think this is one of those times. 

<img width="958" alt="Screenshot 2024-11-21 at 1 26 42 PM" src="https://github.com/user-attachments/assets/97fab302-ce11-45f3-9f99-a037ce74fa01">


https://github.com/pyOpenSci/software-submission/issues/201